weaver/processes/execution.py

```diff
@@ -201,20 +204,21 @@ def execute_process(self, job_id, url, headers=None):
 
                 if execution.isComplete():
                     job.mark_finished()
-                    job.progress = JOB_PROGRESS_EXECUTE_MONITOR_END
                     msg_progress = " (status: {})".format(job_msg) if job_msg else ""
                     if execution.isSucceded():
+                        wps_package.retrieve_package_job_log(execution, job, progress_min, progress_max)
                         job.status = map_status(STATUS_SUCCEEDED)
                         job.status_message = "Job succeeded{}.".format(msg_progress)
-                        wps_package.retrieve_package_job_log(execution, job)
+                        job.progress = JOB_PROGRESS_EXECUTE_MONITOR_DONE
```

Comment:
Why not use `progress_max`? Same comment for below usage of `JOB_PROGRESS_EXECUTE_MONITOR_DONE `.